config.default.yaml

```diff
@@ -43,85 +42,38 @@ crossmatch:
         zMeanPSFMagErr: 1
         yMeanPSFMag: 1
         yMeanPSFMagErr: 1
+        ra: 1
+        dec: 1
     - catalog: Gaia_DR3
       radius: 2.0 # 2 arcseconds
       use_distance: false
       projection:
         _id: 1
-        coordinates.radec_str: 1
         parallax: 1
         parallax_error: 1
         phot_g_mean_mag: 1
         phot_bp_mean_mag: 1
         phot_rp_mean_mag: 1
-    - catalog: SDSS-V_master_20240719
```

Comment:
Are all the catalogs removed here ones we don't need anymore?

Reply:
Basically when I was adding the crossmatch config to the codebase, I added some random catalogs I had on my machine. But, these aren't so relevant to what we do so I remove them here.